bigquery/deploy/README.md

```diff
@@ -56,6 +56,12 @@
     ```
 
 * Run indexer on GKE
+  * We recommend you delete the index, to start from a clean slate.
+    ```
+    kubectl get svc,pods
+    kubectl exec -it ES_CLIENT_POD -- /bin/bash
+    curl -XDELETE EXTERNAL_IP:9200/MY_DATASET
```

Comment:
Have you tested this in a replicated production instance? I could imagine that hooking directly into the pod and deleting the index might cause GKE to try to "autoheal" the pod by re-replicating the index. Can you confirm this isn't the case when we have replicated pods?

Reply:
Confirmed. It would be strange if this wasn't the case.
```
~/data-explorer-indexers (mc/dont-delete-index): kubectl get svc,pods 
NAME                              TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)          AGE
service/elasticsearch             LoadBalancer   10.7.245.5   10.128.0.6    9200:31517/TCP   35d
service/elasticsearch-discovery   ClusterIP      None         <none>        9300/TCP         35d
service/kubernetes                ClusterIP      10.7.240.1   <none>        443/TCP          35d

NAME                             READY     STATUS      RESTARTS   AGE
pod/bq-indexer-4s9j8             0/1       Completed   0          35d
pod/es-client-595585f9d4-kvrrr   1/1       Running     0          35d
pod/es-client-595585f9d4-p4g9g   1/1       Running     0          35d
pod/es-data-78764d56f5-d7fln     1/1       Running     0          35d
pod/es-data-78764d56f5-hnvq5     1/1       Running     0          35d
pod/es-master-78dfb6f7f5-bqkct   1/1       Running     0          35d
pod/es-master-78dfb6f7f5-f79nh   1/1       Running     0          35d
pod/es-master-78dfb6f7f5-jxdpg   1/1       Running     0          35d
~/data-explorer-indexers (mc/dont-delete-index): kubectl exec -it es-client-595585f9d4-kvrrr -- bash
bash-4.4# curl 10.128.0.6:9200/_cat/indices?v
health status index            uuid                   pri rep docs.count docs.deleted store.size pri.store.size
green  open   platinum_genomes 6MKr0N-FTi6z8Ew2h092hQ   5   1         17            0     49.1kb         24.5kb
bash-4.4# curl -XDELETE 10.128.0.6:9200/platinum_genomes
{"acknowledged"10.128.0.6:9200/_cat/indices?v9200/platinum_genomes
bash-4.4# curl 10.128.0.6:9200/_cat/indices?v
health status index uuid pri rep docs.count docs.deleted store.size pri.store.size
bash-4.4# exit
exit
~/data-explorer-indexers (mc/dont-delete-index): kubectl exec -it es-client-595585f9d4-p4g9g -- bash
bash-4.4# curl 10.128.0.6:9200/_cat/indices?v
health status index uuid pri rep docs.count docs.deleted store.size pri.store.size
```